apps/eth_geth_adapter/test/eth_geth_adapter/worker_test.exs

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule EthGethAdapter.WorkerTest do
+  use ExUnit.Case
+  alias Ecto.Adapters.SQL.Sandbox
+  alias Keychain.Repo
+
+  setup tags do
+    :ok = Sandbox.checkout(Repo)
```

Comment:
I think the worker shouldn’t access `Keychain` directly. Instead, let the `EthBlockchain` pass it to the adapter (so the adapter can be self-contained).